vendors/carbonblack.json

```diff
@@ -10,11 +10,5 @@
   "subprocessors": "https://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/downloads/eula/vmware-carbon-black-cloud-sub-processors-jan-2020.pdf",
   "_type": "carbon_black",
   "aliases": ["confer", "bit9 + carbon black", "vmware carbon black"],
-  "cyberDefenseMatrixMap": {
-    "devices": ["protect", "detect", "respond"],
-    "applications": [],
-    "networks": [],
-    "data": [],
-    "users": []
-  }
+  "cyberDefenseMatrixMapping": ["third-party:device:protect", "third-party:device:detect", "third-party:device:respond"]
```

Comment:
this requires additional parsing and it's not ideal. Also there's significant duplicates across each of the 3 strings: `third-party:device:`